### DISCUSSION TITLE
询问显卡，求大佬

### DISCUSSION THREAD

**Discussion by 114514191807343** [share121/inter-knot#1962] [其他]
徐闻一下1800左右的二手显卡，求求了
![微信图片_20240813231152](https://github.com/user-attachments/assets/a7ec2f61-4682-4ea4-bf0c-938bda5ff7a2)

**Comment by cpu644** [comment]
一千八二手？可以蹲个3080 10g或者rx 6800

**Comment by schalesensei843** [comment]
推荐gtx690战术核显卡

**Comment by Buer-Nahida** [comment]
UHD 730 (intel核显)![x](https://i0.hdslb.com/bfs/emote/bf7e00ecab02171f8461ee8cf439c73db9797748.png@40w_40h.avif)
